spec/fields/field_spec.rb

```diff
@@ -0,0 +1,318 @@
+# frozen_string_literal: true
+
+require 'support/origin_info_date_helpers'
+require 'fixtures/imprint_fixtures'
+
+describe ModsDisplay::Field do
+  include ImprintFixtures
+
+  describe '#date_fields' do
```

Comment:
these tests are essentially the same as the ones I removed from the imprint_spec.